_includes/post.html

```diff
@@ -2,15 +2,18 @@
 <article id="{{ post.id | default: post.title | slugify }}"{% if include.excerpt %} class="post-excerpt padding-top-2 border-top-2px margin-top-4 measure-6"{% endif %}>
   <header class="post-meta">
     <p class="post-category site-subheading">
-      {{ post.category | default: post.tags[0] | default: layout.category }}
+      {{ post.tags[0] }}
```

Comment:
All of our categories were "About USWDS," the Updates page category so I default to the first tag and the categories seem to be back working.

Reply:
nice!